meteor-app/imports/api/orders/methods.js

```diff
@@ -0,0 +1,33 @@
+import { Meteor } from 'meteor/meteor'
+import Orders from '/imports/api/orders/orders'
+import log from '/imports/lib/server/log'
+const debug = require('debug')('att:server-methods')
+
+Meteor.methods({
+  'orders.insert'(order) {
+    try {
+      return Orders.insert(order)
+    } catch (e) {
+      log.error({ e })
+      throw new Meteor.Error(500, e.sanitizedError.reason)
+    }
+  },
+  'orders.remove'(id) {
+    try {
+      log.info('removing order: ', id)
+      return Orders.remove({ _id: id })
+    } catch (e) {
+      log.error({ e })
+      throw new Meteor.Error(500, e.sanitizedError.reason)
+    }
+  },
+  'orders.update'(id, orderedParts) {
+    try {
+      log.info('updating order: ', id, orderedParts)
+      return Orders.update({ _id: id }, { $set: { ...orderedParts } })
```

Comment:
this isnt going to work, is orderedParts an object or an array?
the code as it stands will spread an object into the order object, rather than adding extra stuff to the orderedParts key.
https://docs.mongodb.com/manual/reference/operator/update/push/
Hope that makes sense